plugin/src/main/java/com/playmonumenta/scriptedquests/utils/MessagingUtils.java

```diff
@@ -20,61 +27,65 @@
 
 https://ci.md-5.net/job/BungeeCord/ws/chat/target/apidocs/net/md_5/bungee/api/ChatMessageType.html
 */
-import net.md_5.bungee.api.chat.BaseComponent;
-import net.md_5.bungee.api.chat.ClickEvent;
-import net.md_5.bungee.api.chat.ComponentBuilder;
-import net.md_5.bungee.api.chat.HoverEvent;
-import net.md_5.bungee.api.chat.TextComponent;
 // https://www.spigotmc.org/wiki/the-chat-component-api/
 
 public class MessagingUtils {
+	public static final LegacyComponentSerializer LEGACY_SERIALIZER = LegacyComponentSerializer.legacySection();
+	public static final PlainComponentSerializer PLAIN_SERIALIZER = PlainComponentSerializer.plain();
+
 	public static String translatePlayerName(Player player, String message) {
-		return message.replaceAll("@S", player.getName());
+		return (message.replaceAll("@S", player.getName()));
```

Comment:
return doesn't require parentheses. Also note that Adventure lets us replace text with components, like the player selector - but let's save that for another time. Link if you're curious:
https://github.com/KyoriPowered/adventure-text-minimessage/blob/6faaf9c0027fc7399eb43ef30f66dde3f711d4f5/src/main/java/net/kyori/adventure/text/minimessage/Template.java